docker/spec/dependabot/docker/file_parser_spec.rb

```diff
@@ -462,6 +462,34 @@
       end
     end
 
+    context "with a _ in the tag" do
+      let(:dockerfile_fixture_name) { "underscore" }
+
+      its(:length) { is_expected.to eq(1) }
+
+      describe "the first dependency" do
+        subject(:dependency) { dependencies.first }
+        let(:expected_requirements) do
+          [{
+            requirement: nil,
+            groups: [],
+            file: "Dockerfile",
+            source: {
+              registry: "registry-host.io:5000",
+              tag: "17.04_"
+            }
+          }]
+        end
+
+        it "has the right details" do
+          expect(dependency).to be_a(Dependabot::Dependency)
+          expect(dependency.name).to eq("myreg/ubuntu")
+          expect(dependency.version).to eq("17.04_")
```

Comment:
Is it this a real example, or just a simplified version for the sake of testing? Do you have an example of a realworld image tagged like this?

So far we've been trying to support to most widely used versioning schemes out there since there's no official spec for versioning docker tags. But arbitrarily putting an underscore after the version number does not make a lot of sense to me.

Reply:
This is not a real example. The version we're thinking of is non-public, but it's a docker container for which the tag is shaped something like `someRepo_19700101.4`. I can replace the above with that now that I know what shape we're imagining.

Reply:
So, just to confirm, you need to support underscores as suffixes to the version, not as prefixes as originally stated, correct?